xbmc/pvr/windows/GUIWindowPVRBase.cpp

```diff
@@ -359,7 +359,7 @@ void CGUIWindowPVRBase::SetInvalid()
 bool CGUIWindowPVRBase::CanBeActivated() const
 {
   // No activation if PVR is not enabled.
-  return CSettings::GetInstance().GetBool(CSettings::SETTING_PVRMANAGER_ENABLED);
```

Comment:
it should have tested if PVRManager was running, not if it was enabled. On my list ...